debtransform

```diff
@@ -270,6 +278,13 @@ sub dotar_quilt {
     unlink($changelog_file_path);
     rename($changelog_file_path_patched, $changelog_file_path);
 
+    # Copy to temp dir if not already copied
+    if ((index ${tar_for_concatination_path}, ${tempdirname}) == -1){
+      print "Copy '${tar_for_concatination_path}' to '${tempdirname}' for patching\n";
+      system("cp \"$decompress_src\" \"$decompress_dst\"") == 0 || die("Can not copy archive '$decompress_src': $!\n");
```

Comment:
Call it with an list as param.

Reply:
In all 'system' function calls, the strings have been replaced with an argument list. In places where pipes were, 'system' function calls have been replaced with IPC::Run.